test/jest/__mock__/stripesComponents.mock.js

```diff
@@ -33,9 +48,23 @@ jest.mock('@folio/stripes/components', () => ({
       {children}
     </div>
   )),
-  ConfirmationModal: jest.fn(() => <div>ConfirmationModal</div>),
+  ConfirmationModal: jest.fn(({ heading, confirmLabel, cancelLabel, onConfirm, onCancel }) => (
```

Comment:
I do not see the usage of this component inside `PickupServicePointFilter`. Do we really need to mock it?